Resources/templates/CommonAdmin/NestedListTemplate/NestedListBuilderTemplate.php.twig

```diff
@@ -21,3 +21,23 @@
         {{ echo_include(builder.namespacePrefixForTemplate ~ bundle_name ~ ':' ~ builder.BaseGeneratorName ~ 'List:results.html.twig') }}
     </div>
 {{ echo_endblock() }}
+
+{{ echo_block('modals') }}
+<div id="confirmModal" class="modal fade" role="dialog">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <form role="form" action="#" method="post" data-ajax="false">
+        <input type="hidden" name="_csrf_token" value=""/>
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal">&times;</button>
+          <h4 class="modal-title"></h4>
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-default" data-dismiss="modal">{{ echo_trans('action.custom.cancel') }}</button>
+          <button type="submit" class="btn btn-primary">{{ echo_trans('action.custom.confirm') }}</button>
+      </div>
+      </form>
+    </div>
+  </div>
+</div>
+{{ echo_endblock() }}
```

Comment:
This looks like a duplication of the code in the ListBuilderTemplate. So, it would be better to create a separate template which is included here: also makes it easier to customize :)